Resources/Prototypes/DeltaV/Entities/Clothing/Eyes/glasses.yml

```diff
@@ -12,3 +12,33 @@
     modifiers:
       coefficients:
         Caustic: 0.85
+
+- type: entity
+  parent: [ClothingEyesBase, ShowSecurityIcons, ShowMedicalIcons, BaseRestrictedContraband]
```

Comment:
if you are going to use contraband use base security contraband

Reply:
![image](https://github.com/user-attachments/assets/475a3c7c-0e1b-45f8-82c1-6d4fca3d7fb7)
The normal security glasses also use the "Restricted Contraband"

Reply:
there is also no "BaseSecurityContraband"